resources/views/settings/ldap.blade.php

```diff
@@ -91,6 +91,53 @@
                             </div>
                         </div>
 
+                        <!--  Default LDAP Permissions Group Select -->
+
+                        <div class="form-group{{ $errors->has('group') ? ' has-error' : '' }}">
+                            <div class="col-md-3">
+                                {{ Form::label('ldap_default_group', trans('admin/settings/general.ldap_default_group')) }}
+                            </div>
+
+                            <div class="col-md-9">
+
+                                @if ($groups->count())
+                                    @if ((Config::get('app.lock_passwords') || (!Auth::user()->isSuperUser())))
+                                        <ul>
+                                            @foreach ($groups as $id => $group)
+                                                {!! '<li>'.e($group).'</li>' !!}
+                                            @endforeach
+                                        </ul>
+
+
+                                        <span class="help-block">{{ trans('admin/users/general.group_memberships_helpblock') }}</span>
+                                    @else
+                                        <div class="controls">
+                                            <select
+                                                    name="ldap_default_group"
+                                                    aria-label="ldap_default_group"
+                                                    id="ldap_default_group"
+                                                    class="form-control"
+                                            >
+
+                                                @foreach ($groups as $id => $group)
+                                                    <option value="{{ $id }}">
```

Comment:
This looks good - but we don't have a "blank" option. If an admin doesn't want to assign a group, or wants to remove that setting, we don't have a "Select Group" option with no value.

Reply:
blank option added